Android/app/src/main/AndroidManifest.xml

```diff
@@ -119,8 +121,11 @@
             android:grantUriPermissions="true">
             <meta-data
                 android:name="android.support.FILE_PROVIDER_PATHS"
-                android:resource="@xml/provider_paths"/>
+                android:resource="@xml/provider_paths" />
         </provider>
+
+        <activity android:name=".ProfileActivity"
+            android:parentActivityName=".login.LoginActivity"/>
```

Comment:
Why are you making other refactoring changes. You just need to add this ProfileActivity

Reply:
Sorry forgot to remove